Jenkinsfile

```diff
@@ -6,5 +6,6 @@
 buildPlugin(useContainerAgent: true, configurations: [
```

Comment:
As a first step, you might want to switch using a VM agent (instead of a container agent) as per https://github.com/jenkins-infra/helpdesk/issues/4047#issuecomment-2060457602

Reply:
I've committed that to the origin branch:
https://github.com/jenkinsci/winp/blob/ci_windows/Jenkinsfile

Reply:
Thanks. If we use a full VM image, then probably adding VS directly inside it is much better than expecting every pipeline to need to install this. I asked for this on open infra ticket, and I hope this will be accepted :)